src/components/Button/Button.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
Button component should be listed in `src/index.js`

```js
export { default as Dummy } from './components/Dummy';
export { default as Button } from './components/Button';
```

Reply:
Already fixed